upload-firmware-guide.md

```diff
@@ -52,13 +52,19 @@ _Note: you will most likely have to click "Click here to download manually". If
 
 ## 5. Clone the firmware project
 
-1. Open Powershell window or Command Prompt window in the folder where you want to clone the firmware project, and execute the following command: `git clone https://github.com/SlimeVR/SlimeVR-Tracker-ESP.git SlimeVR-Tracker-ESP-main`.
-1. Once the project is cloned, open the project in Visual Studio Code:
```

Comment:
I'd suggest to keep these steps, because if you open PIO project for the first time, it may not be recognized by PIO and the PIO functionality won't appear.

Reply:
I tested all of this, it's the easiest way to do this and there was a lot of confusion about how to get stuff from git on the command line